ova-compose/ova-compose.py

```diff
@@ -1271,6 +1273,9 @@ def main():
             params[k] = yaml.safe_load(v)
         elif o in ['--tar-format']:
             tar_format = a
+        elif o in ['--vmdk-convert']:
+            global VMDK_CONVERT
+            VMDK_CONVERT = a
```

Comment:
I think this is needed otherwise 'VMDK_CONVERT' is local to this function only and where we use it is in different function. It is failing for me.
Can you remove vmdk-convert binary from your system then try. I think that should not work.

Reply:
Hm, you are right. I wasn't aware of this quirk - when reading, it works, but when assigning a value it creates a local variable instead of setting the global one. https://stackoverflow.com/questions/423379/how-to-use-a-global-variable-in-a-function .